src/main/java/no/nav/familie/integrasjoner/oppgave/OppgaveController.kt

```diff
@@ -133,8 +137,11 @@ class OppgaveController(private val oppgaveService: OppgaveService) {
         @Parameter(description = "Settes til true hvis man ønsker å flytte en oppgave uten å ta med seg mappa opp på oppgaven. Noen mapper hører spesifikt til en enhet, og man får da ikke flyttet oppgaven uten at mappen fjernes ")
         @RequestParam(name = "fjernMappeFraOppgave")
         fjernMappeFraOppgave: Boolean,
+        @Parameter(description = "Settes dersom man ønsker å sjekke at oppgaven man har tilgjengelig er riktig versjon.")
```

Comment:
Er det "for lett" å si her at - `vil feile med 409 Conflict dersom versjonen ikke stemmer overens med oppgavesystemets versjon` 
?

Reply:
Hmm ja just det, jeg tror kanskje ikke vi kaster videre 409, det må nog også legges inn i `ApiExceptionHandler`

Reply:
Lagt til at 409 kastes om oppdater returnerer 409 og endret beskrivelsen til Charlie sitt forslag